dotCMS/src/integration-test/java/com/dotcms/rest/api/v1/apps/AppsResourceTest.java

```diff
@@ -144,30 +116,31 @@ private FormDataMultiPart createFormDataMultiPart(final String fileName,
      * @throws DotSecurityException
      */
     @Test
-    public void Test_Create_app_descriptor_Then_Create_App_Integration_Then_Delete_The_Whole_App() {
+    public void Test_Create_app_descriptor_Then_Create_App_Integration_Then_Delete_The_Whole_App()
+            throws IOException {
 
         final Host host = new SiteDataGen().nextPersisted();
-
-        final SortedMap<String, ParamDescriptor> paramMap = ImmutableSortedMap.of(
-                "p1", ParamDescriptor.newParam("v1", false, Type.STRING, "label", "hint", true),
-                "p2", ParamDescriptor.newParam("v2", false, Type.STRING, "label", "hint", true),
-                "p3", ParamDescriptor.newParam("v3", false, Type.STRING, "label", "hint", true)
-        );
+        final AppDescriptorDataGen dataGen = new AppDescriptorDataGen()
+                .stringParam("p1", false,  true)
+                .stringParam("p2", false,  true)
+                .stringParam("p3", false,  true)
+                .withName("any")
+                .withDescription("demo")
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String literal "demo" appears 9 times in this file; the first occurrence is on line 136](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5358201)